requirements.txt

```diff
@@ -0,0 +1,7 @@
+aiohttp
+fastapi
+loguru >= 0.7.2
+transformers >= 4.36.0
+sentencepiece
+uvicorn[standard]
+vllm>=0.2.5
```

Comment:
pin the version of vllm as we are performing benchmarks. What if the reader installs vllm by this file and ends up with vllm 0.3.0? The result will not be reproducable then.

Only when building libraries we need to set a loose dependency range.